app/javascript/dashboard/components/CustomBrandPolicyWrapper.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+import { useStoreGetters } from 'dashboard/composables/store';
+import { computed } from 'vue';
+const props = defineProps({
+  showOnCustomBrandedInstance: {
```

Comment:
There's a mismatch between the prop name in the defineProps and where it's used in the computed property. The prop is defined as `showOnCustomBrandedInstance`, but it's referred to as `displayOnCustomBrandedInstance` in the computed property.

Reply:
I've enabled the eslint rules for the same.

Reply:
Good catch!